app/src/main/java/com/toggl/domain/mappings/Settings.kt

```diff
@@ -1,23 +1,31 @@
 package com.toggl.domain.mappings
 
+import com.toggl.architecture.Loadable
 import com.toggl.domain.AppAction
 import com.toggl.domain.AppState
+import com.toggl.models.domain.User
 import com.toggl.settings.domain.SettingsAction
 import com.toggl.settings.domain.SettingsState
 
-fun mapAppStateToSettingsState(appState: AppState): SettingsState =
-    SettingsState(
+fun mapAppStateToSettingsState(appState: AppState): SettingsState? {
+    val user = (appState.user as? Loadable.Loaded<User>)?.value ?: return null
+    return SettingsState(
+        user,
```

Comment:
I think this would read much better without the parenthesis

```suggestion
    val user = appState.user as? Loadable.Loaded<User> ?: return null
    return SettingsState(
        user.value,
```